cmd/verify.go

```diff
@@ -34,3 +34,8 @@ func showVerifyUsage() {
 	fmt.Printf(verifyf, exeName())
 	os.Exit(1)
 }
+
+// Verify is a public function to be imported and execute Verify function from internal/verify/verify.go
```

Comment:
This is going to be part of the public API, so it would be nice to align with Go doc conventions:

```suggestion
// Verify checks whether a given Rancher Prime image is signed based on the Cosign Signature spec.
// The same extents to CNCF images within the Rancher ecosystem.
//
// Upstream documentation:
// https://github.com/sigstore/cosign/blob/main/specs/SIGNATURE_SPEC.md
```

For a more long term change, I think it make more sense to remove this `func`, and instead move `internal/verify` to `pkg/verify`.